nextflow.config

```diff
@@ -74,15 +72,15 @@ try {
 profiles {
     debug { process.beforeScript = 'echo $HOSTNAME' }
     conda {
-        params.enable_conda    = true
+        conda.enabled          = true
```

Comment:
what does this do by changing from a parameter to true by default? is it so the CI tests will run?

Reply:
this is another nf-core change for conda declarations